feat: steward aip (#423)
aave/aip: content/aips/CAPS_PLUS_RISK_STEWARD.md

---
title: Caps Plus Risk Steward
author: BGD Labs (@bgdlabs)
shortDescription: Grant RISK_ADMIN roles to the CapsPlusRiskStewards on the respective networks
discussions: https://governance.aave.com/t/bgd-risk-steward-phase-1-capsplusrisksteward/12602
created: 2023-05-23
---

## Simple Summary

This governance proposal grants the `RISK_ADMIN` role to the `CapsPlusRiskStewards`.

> Once the `CapsPlusRiskStewards` has received the `RISK_ADMIN` role, the `RISK_COUNCIL` can adjust asset borrow/supply caps upwards without voting overhead. The `RISK_COUNCIL` is controlled by the DAOs risk-providers (Gauntlet, Chaos Labs).

Reviewer: ```suggestion
Once the `CapsPlusRiskStewards` has received the `RISK_ADMIN` role, the `RISK_COUNCIL` can adjust asset borrow/supply caps upwards without voting overhead. The `RISK_COUNCIL` is a Gnosis Safe controlled by the DAOs risk-providers (Gauntlet, Chaos Labs).
```

## Motivation

During the last months, for the broad majority of proposals regarding caps updates, the support of the community has been almost unanimous, with 1 or 2 exceptions.
The rationale of this is pretty clear: caps updates usually simply depend on the “green light” from the risk providers (Gauntlet, Chaos Labs)
As both providers have historically been in agreement with the updates proposed, the community has just followed that recommendation by supporting on-chain.

> Therefore, it seems that the community has important voting overhead in this scenario, which could be almost completely automated and simplified. It is quite common to have a proposal with unanimous support on day 1 of voting but given all other time considerations, execution is 5 days after.

Reviewer: ```suggestion
Therefore, it seems that the community has important voting overhead in this scenario, which could be almost completely automated and simplified. It is quite common to have a proposal with unanimous support on day 1 of voting but given all other time considerations, execution is 3 to 5 days after.
```

This proposal passes permissions to the `CapsPlusRiskSteward` introduced in this [governance forum thread](https://governance.aave.com/t/bgd-risk-steward-phase-1-capsplusrisksteward/12602).

> After execution the proposal the `RISK_COUNCIL` will be able to perform cap increases trough the `CapsPlusRiskStewards` under the following conditions:

Reviewer: ```suggestion
After execution, the `RISK_COUNCIL` will be able to perform cap increases through the `CapsPlusRiskStewards` under the following conditions:
```
